requirements.txt

```diff
@@ -1,3 +1,11 @@
-# Trick for ReadTheDocs to install all requirements:
+Flask>=0.9
```

Comment:
these shouldn't be necessary.

Reply:
I thought I had changed that back.  I'm reverting it